### ISSUE TITLE
The map on the mobile devices does not work.

### ISSUE THREAD

**Issue by where2help-fsw** [where2help/where2help#221]
When you open an event with mobile, the map is not displayed on the mobile device. Tried using Android and Apple.

**Comment by ramonh** [comment]
Hi @where2help-fsw! I just tried logging in on Chrome and Safari on iOS as a volunteer and was able to see the map. I'm curious, were you logged in as an NGO or a volunteer?

**Comment by klappradla** [comment]
Hey @where2help-fsw can you please provide a link to the specific event and/or a screenshot?
Is the behaviour occurring for **every** event, or just one specific one?

**Comment by where2help-fsw** [comment]
Hi,
we tried it on ipad and iphone using safari. I am sorry but I don’t know which browser my collegue uses. I was logged in as a volunteer and we tried it about 5 times within 2 hours. If ist’s working now: great and sorry for any inconvenience ☺
Thanks a lot, Antonia

Von: Ramon Huidobro [mailto:notifications@github.com]
Gesendet: Freitag, 04. November 2016 15:40
An: where2help/where2help
Cc: Fonds Soz. Wien where2help; Mention
Betreff: Re: [where2help/where2help] The map on the mobile devices does not work. (#221)


Hi @where2help-fsw<https://github.com/where2help-fsw>! I just tried logging in on Chrome and Safari on iOS as a volunteer and was able to see the map. I'm curious, were you logged in as an NGO or a volunteer?
